docs/architecture/self-hosted/authentication/databricks-oauth.md

```diff
@@ -61,6 +63,14 @@ You only need to perform this authentication periodically, depending on the OAut
 
 :::
 
+#### Run Job configuration
+
+When configuring a Job for SQL, the the Service Principal specified during Fabric creation will be used instead of the user specified under Run As.
```

Comment:
Adhoc job runs will be run as the "user" themselves. Scheduled jobs will run as the Service Principal specified in the fabric. A warning will come up in Airflow/Databricks Job IDE if any SQL fabric being used doesn't contain Service Principal OAuth Credentials yet

Reply:
Done.